src/main/java/net/minecraftforge/client/event/RenderPlayerEvent.java

```diff
@@ -53,15 +53,15 @@ public RenderPlayerEvent(Player player, PlayerRenderer renderer, float partialTi
     @Cancelable
     public static class Pre extends RenderPlayerEvent
     {
-        public Pre(Player player, PlayerRenderer renderer, float tick, PoseStack stack, MultiBufferSource buffers, int light) {
-            super(player, renderer, tick, stack, buffers, light);
+        public Pre(Player player, PlayerRenderer renderer, float partialTick, PoseStack poseStack, MultiBufferSource multiBufferSource, int packedLight) {
+            super(player, renderer, partialTick, poseStack, multiBufferSource, packedLight);
```

Comment:
I've seen `MultiBufferSource` in some previous files named as `bufferSource`. Is the discrepancy intentional, or should either `bufferSource` or `multiBufferSource` be applied to all files?

Reply:
This particular parameter was named to line up the field in `RenderPlayerEvent` constructor it is passed to. My personal preference is `bufferSource` and could be changed back leaving the field/method name changes pending if others prefer this too.

Reply:
In the interests of keeping the field and parameters matching, we'll keep this as `multiBufferSource` then.